src/elements/post/footer/footer.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { createElement, ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  Avatar,
+  Container,
+  Grid,
+  Typography,
+  Divider,
+  makeStyles
+} from '@material-ui/core'
+
+const useStyles = makeStyles((theme) => ({
+  divider: {
+    margin: theme.spacing(5, 0)
+  },
+  image: {
+    width: 180,
+    height: 180
+  },
+  link: {
+    color: theme.palette.text.primary,
+    textDecoration: 'underline',
+    cursor: 'pointer'
+  },
+  author: {
+    textAlign: 'center',
+    marginBottom: theme.spacing(4)
+  },
+  heading: {
+    textAlign: 'center'
+  }
+}))
+
+type AuthorAvatar = {
+  url: string
+}
+
+type Author = {
+  /** Avatar */
+  avatar: AuthorAvatar
+  /** Author Name */
+  name: string
+  /** Author Biography */
+  description: string
+}
+
+type AuthorNode = {
+  node: Author
+}
+
+export type PostFooterProps = {
+  /**
+   * Author
+   */
+  author?: AuthorNode
+  AuthorLink?: (
+    props: React.DetailedHTMLProps<
+      React.AnchorHTMLAttributes<HTMLAnchorElement>,
+      HTMLAnchorElement
+    >
+  ) => ReactElement
+}
+
+export function PostFooter({
+  author,
+  AuthorLink = (props) => createElement('a', props)
+}: PostFooterProps) {
+  const { t } = useTranslation('post-footer')
+  const classes = useStyles()
+  return (
+    <>
+      <Container maxWidth="sm">
+        <Divider className={classes.divider} />
+        <Grid container justify="center" spacing={1}>
+          <Grid item sm={5}>
+            <Avatar
+              alt={author.node.name}
+              src={author.node.avatar.url}
+              className={classes.image}
+            />
+          </Grid>
+          <Grid item sm={7}>
+            <Typography className={classes.author} variant="h5">
+              <AuthorLink className={classes.link} data-testid="author-link">
+                {author.node.name}
+              </AuthorLink>
```

Comment:
add `import { Link } from '@material-ui/core'`

then do something like:

```suggestion
          <Link component={AuthorLink} href={author.node.slug}>
            {author.node.name}
          </Link>
```

See PostCard premiere variant for details.

Reply:
Kept the className